Updating doc inconsistencies (#95)
infrawatch/documentation: doc-Service-Telemetry-Framework/modules/proc_configuring-red-hat-openstack-platform-overcloud-for-stf.adoc

[[creating-stf-connectors]]
== Creating the {ProjectShort} connection file

> Create a file in which you configure the connection of the {MessageBus} on the overcloud to the {ProjectShort} deployment, and enable collection of events and storage of the events in {ProjectShort}.

Reviewer: ```suggestion
Create a file that contains the connection configuration of the {MessageBus} for the overcloud to the {ProjectShort} deployment, and enable collection of events and storage of the events in {ProjectShort}.
```
